web/modules/custom/usa_twig_vars/usa_twig_vars.module

```diff
@@ -112,8 +112,11 @@ function usa_twig_vars_preprocess(&$variables, $hook) {
 
 /**
  * Hide menu items with page type set to Standard Page - Nav Hidden.
+ *
+ * @todo Check if this menu/hook is used anywhere
```

Comment:
This is a head-scratcher (whether usa_twig_vars_preprocess_menu__sidebar_first is used anywhere). 

Based on the comment, it would have been meant to hide pages like `/page-error` from the menu on their parent pages, e.g., `/contact-us`, but this little bit of code doesn't seem to do anything like that. 

I don't have an answer on this one.

Reply:
I didn't see an ovious answer. I suspect menu__sidebar_first is for a region or block that may not exist anymore